internal/runtime/runtime_factory.go

```diff
@@ -128,8 +128,8 @@ func supportedModifierTypes(mode string) []string {
 		return []string{"nvidia-hook-remover", "mode"}
 	case "csv":
 		// For CSV mode we support mode and feature-gated modification.
-		return []string{"nvidia-hook-remover", "mode", "feature-gated"}
+		return []string{"nvidia-hook-remover", "feature-gated", "mode"}
```

Comment:
Question -- why does the ordering matter?

Reply:
In the CSV case, I don't think it's strictly needed, but the intent is to ensure that the `ldconfig` hook that is injected as part of the `mode` modifications is *after* the hook to configure the compat libraries. Note that since we may also add libraries in the `feature-gated` modifiers, we may have been missing these libraries from the ldcache in the past.